### ISSUE TITLE
PATCH: 'Server error' is returned instead of 'Bad request'

### ISSUE THREAD

**Issue by aliaksiejmaroz** [simpleidserver/SimpleIdServer#357]
If you send something like this
```
{
    "schemas": [
        "...."
    ],
    "Operations": [
        {
            "op": "replace",
            "path": "active",
            "value": 234
        }
    ]
}
```
**server error (500)** is returned.

There should be **validation** of the **value** for **PATCH** method.
In case of not boolean value, **400** status code with _“invalidValue“_ error in the body should be returned. 

![image](https://user-images.githubusercontent.com/10138979/206183435-3dc5c246-7a81-466c-a786-565c2d3fadea.png)

**Comment by simpleidserver** [comment]
The bug is fixed in the branch "release/v2.0.22".

KR,

SID

**Comment by aliaksiejmaroz** [comment]
Great! Thanks! When do you plan to release v2.0.22?

**Comment by simpleidserver** [comment]
The nuget package will be released next week, when the ticket #343 is finished.
